runtime/tests/checker/interface_test.go

```diff
@@ -1834,6 +1836,68 @@ func TestCheckContractInterfaceFungibleTokenConformance(t *testing.T) {
 	}
 }
 
+func BenchmarkContractInterfaceFungibleToken(b *testing.B) {
```

Comment:
Nice, did these show good results from the change?

Reply:
A bit, as there's less work done now, but not much.